client/public/index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8" />
-  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  <link rel="icon" href="https://amplify-austin.s3.us-east-2.amazonaws.com/amplify_austin.png" />
```

Comment:
is this supposed to be the title/header image? Because youve replaced the favicon instead

Reply:
No, I was replacing the favicon so it matched our logo